inc/ui/namespace.php

```diff
@@ -31,16 +32,26 @@ function bootstrap() {
  * to the admin bar.
  */
 function register_menu() {
-	$hook = add_submenu_page(
+	// Add top level page.
+	$hook = add_menu_page(
```

Comment:
Default position for `add_menu_page()` is the bottom of the list. There wasn't a position specified, but the ticket does say top level. I don't think it would be found if it was under the tools menu.

Reply:
Sorry I meant adjacent to Tools rather than under that menu item. We won't always think of every little detail during refinements so there's room for us to use some personal judgment too if you form opinions / new ideas while working on something. If you decide the acceptance criteria could be better or are missing something part way through I'm always happy to chat through it.